app/templates/application.hbs

```diff
@@ -1,18 +1,14 @@
-{{set-body-class "deprecations deprecations_v2 responsive"}}
+<EsHeader>
+  <form class="search-form">
+    <SearchInput />
```

Comment:
We need to put a label in here for the search input, even if we visually hide the label.